.github/workflows/android-appcenter.yml

```diff
@@ -3,15 +3,17 @@ name: Android AppCenter Tests
 on:
   workflow_dispatch:
   push:
-    branches: [ main ]
-    paths: 
-      - 'demo/android/Activity'
+    branches: [ main, 1.1 ]
+    paths:
+      - 'binding/android/cobra/build.gradle'
+      - 'demo/android/Activity/cobra-activity-demo-app/build.gradle'
       - '!demo/android/Activity/README.md'
     
   pull_request:
-    branches: [ main ]
+    branches: [ main, 1.1 ]
     paths: 
-      - 'demo/android/Activity'
+      - 'binding/android/cobra/build.gradle'
```

Comment:
Would we not also want to trigger if any of the source files change? Why just the `build.gradles`?

Reply:
Same for iOS

Reply:
fixed for android

Reply:
The appcenter tests run the unit tests that are embbeded in the demo. Really they just test the Android binding, not the demo code, which is why we just wait for the package referenced in the `build.gradle` to update.